lib/mindwendel_web/templates/layout/root.html.leex

```diff
@@ -11,8 +11,8 @@
   </head>
 
   <body class="bg-light">
-    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white border-bottom mb-2" style="height:50px;">
-      <div class="container">
+    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white border-bottom mb-2">
+      <div class="container-fluid">
```

Comment:
Adding `container-fluid` allowed the navbar to use the full width. Was this done on purpose? I do not mind it, but maybe we should do this in a separate pull request because it could also have some implications for other pages, e.g. `home.leex`.

Reply:
Woops, good point. I change it back to container.